plugins/source/aws/policies/queries/acm/certificates_expired.sql

```diff
@@ -0,0 +1,14 @@
+insert into aws_policy_results
+select
+    :'execution_time' as execution_time,
+        :'framework' as framework,
+        :'check_id' as check_id,
+        'Ensure that all the expired SSL/TLS certificates stored in AWS IAM are removed (Automated)' as title,
+    account_id,
+    arn AS resource_id,
+    case when
+            not_after < NOW() AT TIME ZONE 'UTC'
+        then 'fail'
+        else 'pass'
+        end as status
+FROM aws_acm_certificates
```

Comment:
This check is not checking ACM certs, it is checking certs in IAM

as returned by `aws iam list-server-certificates` . I am not sure if we support this resource though. We might get the information as a field in the `AccountReport`

Reply:
^should be here and yes from AccountReport https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/docs/tables/aws_accounts.md.